functions/src/graphql/context/data/menuitem.repository.ts

```diff
@@ -0,0 +1,41 @@
+import { Service, Inject } from "typedi";
+import { make } from "../../../helpers/make";
+import { MenuItem } from "../../schema";
+import {store} from "./store.repository"
+import { CoreContext } from "../core.context";
+
+@Service()
+export class MenuItemRepository {
```

Comment:
Let's ditch this entirely for now. I want to make it simpler for us to fetch info and will send an update on how to do that.